src/algebra/ordered_group.lean

```diff
@@ -89,6 +89,13 @@ instance ordered_comm_group.has_exists_mul_of_le (α : Type u)
   has_exists_mul_of_le α :=
 ⟨λ a b hab, ⟨b * a⁻¹, (mul_inv_cancel_comm_assoc a b).symm⟩⟩
 
+@[to_additive] instance [has_inv α] : has_inv (order_dual α) := ‹has_inv α›
```

Comment:
For good measure, can you add
```suggestion
@[to_additive] instance [has_inv α] : has_inv (order_dual α) := ‹has_inv α›
@[to_additive] instance [has_div α] : has_div (order_dual α) := ‹has_div α›
```

Reply:
These two maybe belong in `ordered_monoid` next to `order_dual.div_inv_monoid` (or vice versa)

Reply:
Moved `div_inv_monoid` to `ordered_group`, added a few more instances.

Reply:
Thanks, feel free to merge when CI is happy.